API.md

```diff
@@ -0,0 +1,106 @@
+## Models
+
+### Blueprint
+TBD
+
+### Header
+
+```
+{
+	type:      'header';
+	id:        number;
+	sourceUrl: string; // URL of the original page from which the header was created.
+	// More fields TBD.
+}
+```
+### Footer
+
+```
+{
+	type:      'footer';
+	id:        number;  
+	sourceUrl: string; // URL of the original page from which the footer was created.
+	// More fields TBD.
+}
+```
+### BlogPost
+
+```
+{
+	type:          'blog-post';
+	id:            number;  
+	transformedId: number;      // Id of the corresponding transformed post
+	sourceUrl:     string;      // URL of the source page from which this blog post was created.
+	date: {
+		type:     'date';
+	    original: string; // original HTML
```

Comment:
I think we should rename `original` to `raw`. Original indicates a valid previous value, but Raw holds the essence correctly i.e. its raw and requires processing.